boards/bluepill-stm32f030c8/include/periph_conf.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2020 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_bluepill-stm32f030c8
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the bluepill-stm32f030c8 board
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @author      Benjamin Valentin <benpicco@googlemail.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+/* This board provides a LSE */
+#ifndef CONFIG_BOARD_HAS_LSE
+#define CONFIG_BOARD_HAS_LSE        1
+#endif
+
+/* This board provides an HSE */
+#ifndef CONFIG_BOARD_HAS_HSE
+#define CONFIG_BOARD_HAS_HSE        1
+#endif
+
+#include "periph_cpu.h"
+#include "f0/cfg_clock_default.h"
```

Comment:
```suggestion
#include "clk_conf.h"
```

This needs to be updated to the clock config cleanup.